### ISSUE TITLE
Allow attribute bindings to work multiline

### ISSUE THREAD

**Issue by samccone** [Polymer/polymer#2560]
Currently in polymer if you have a computed binding such as

```html
<div foo="[[wow(a, v, b)]]"></div>
```

Things are dandy, however if you use a return to break up multiple arguments

```html
<div foo="[[wow(a, 
v, 
b)]]"></div>
```

Things no longer work, perhaps they should?

**Comment by samccone** [comment]
PR to resolve this issue https://github.com/Polymer/polymer/pull/2486

**Comment by samccone** [comment]
**cc** @tjsavage @kevinpschaaf

**Comment by TimvdLippe** [comment]
@samccone I think this was fixed in #2486

**Comment by azakus** [comment]
Fixed in #2486
